src/device/prims_ll.h

```diff
@@ -260,13 +260,23 @@ class Primitives<T, RedOp, Fan, Direct, ProtoLL, P2p, isNetOffload>:
 
   __device__ void storeLL(union ncclLLFifoLine* dst, uint64_t val, uint32_t flag) {
 #if defined(__HIP_PLATFORM_AMD__) || defined(__HIPCC__)
+#if defined(__gfx950__)
```

Comment:
Don't we also need to check if `HIP_HOST_UNCACHED_MEMORY` is supported - otherwise, this will always execute for gfx950?